tern/analyze/common.py

```diff
@@ -249,6 +250,24 @@ def add_base_packages(image_layer, binary, shell):
             shell = constants.shell
         # get all the packages in the base layer
         pkg_dict, invoke_msg, warnings = collate_list_metadata(shell, listing)
+
+        if listing.get("pkg_format") == "deb":
+            pkg_dict["licenses"] = list()
+            pd = os.path.join(
+                rootfs.mount_dir, constants.temp_folder, constants.mergedir)
+            for copyright in pkg_dict["copyrights"]:
+                cp = '{0}{1}'.format(pd, copyright.strip())
+                d = DebianCopyright.from_file(cp)
```

Comment:
I wonder if DebianCopyright has a parser that would read data from `pkg_dict["copyright"]` instead. If reading it from file is more accurate, we may want to create a separate python script for it.

Reply:
I think it is a good idea to abstract this function call using a custom module/script. It may also help in a situation where we may want to add more filtration for license text in the future.

Reply:
That is the rub. Typically, a package has a declared license (see rpm, pip, etc). In Debian's case, their licenses are listed in the copyright text of the package. So this is really post processing of the licenses in a Debian specific copyright file. This is why I asked if `debut` can parse any individual copyright text and get a list of licenses for it. If it can, you can just write a function to do it and call it here.

Reply:
@nishakm how to handle `Unable to import 'debut.copyright'` lint error here?

Reply:
you would have to install tern first in your virtual environment `pip install -e.[dev]`.

Reply:
I am sorry but I was referring circleci linting errors.

Reply:
You will have to add `debut` to `requirements.txt` and `requirements.in`.

Reply:
@nishakm, I have updated the code with requested changes. Please take a look and share your feedback.